scripts/exgfs_wave_prdgen_bulls.sh

```diff
@@ -177,25 +177,25 @@
      echo "${msg}"
      set_trace
      echo "${RUN} wave prdgen ${date} ${cycle} : error in formbul."
-     err=5;export err;err_chk
-     exit ${err}
+     export err=5
+     err_chk
```

Comment:
Similar question about this section...should lines 168/178 be removed and lines 170 through 179 compressed into a message for `err_chk`?